html-forms/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <label for="email">Email:</label>
+
+      <input type="email" id="email" size="30" required>
+      <form action="http://www.example.com/profile.php">
+      <p>Why is coding cool?</p>
+<select name="reasons">
+  <option value="select an option">Select an option</option>
+  <option value="Because it's interesting">Because it's interesting</option>
+  <option value="Because it's fun">Because it's fun</option>
+  <option value="Because it's powerful">Because it's powerful</option>
+</select>
+    </form>
+
+    <form action="http://www.example.com/profile.php">
+    <p>Favorite Language (choose one)
```

Comment:
you don't want to use `p` tags in your form, this is where you want to use `div` elements instead